optimum/habana/transformers/training_args.py

```diff
@@ -487,7 +492,9 @@ def __post_init__(self):
 
         # if training args is specified, it will override the one specified in the accelerate config
         mixed_precision_dtype = os.environ.get("ACCELERATE_MIXED_PRECISION", "no")
-        if self.bf16:
+        if self.fp8:
+            mixed_precision_dtype = "fp8"
```

Comment:
It is risky to overwrite mixed_precision_dtype to "fp8", let's figure out another way of doing this.

Reply:
As discussed, we will proceed with this implementation.